x-pack/plugins/security_solution/public/flyout/entity_details/host_right/hooks/use_observed_host_fields.test.ts

```diff
@@ -10,7 +10,7 @@ import { useObservedHostFields } from './use_observed_host_fields';
 import { mockObservedHostData } from '../../mocks';
 import { TestProviders } from '../../../../common/mock';
 
-describe('useManagedUserItems', () => {
+describe('useObservedHostFields', () => {
   it('returns managed user items for Entra user', () => {
```

Comment:
nit: maybe update the text below to reference host too?
 `it('returns managed host items for Entra host', () => {`

Reply:
done!